open3d_slam_ros/launch/mapping_rosbag.launch

```diff
@@ -14,10 +14,7 @@
 
 	<arg name="play_delay" default="0.4" />
 	<arg name="play_rate" default="1.0" />
-	<arg name="use_sim_time" default="true" />
```

Comment:
all of these are repeated

Reply:
bag_folder_path is not repeated.

line 24 should be 

```
	<arg name="rosbag_full_path" 
		default="$(arg bag_folder_path)/$(arg bag_filename)" /> 
```


instead of 


```
	<arg name="rosbag_full_path" 
		default="$(find open3d_slam_ros)/data/$(arg bag_filename)" /> 
```

Reply:
But thnk you for spotting that too